mockingbird/processor/src/test/kotlin/com/anthonycr/mockingbird/processor/MockingbirdSymbolProcessorTest.kt

```diff
@@ -21,7 +22,10 @@ class MockingbirdSymbolProcessorTest {
         inheritClassPath = true
         verbose = false
         sources = list
-        symbolProcessorProviders = listOf(MockingbirdSymbolProcessorProvider())
+        symbolProcessorProviders += MockingbirdSymbolProcessorProvider()
```

Comment:
This is redundant with the block below and is somehow breaking compilation. I would guess that it might be trying to run the processor twice and wiping out the original output. Whatever, it is, this line needs to be removed.